Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+WORKDIR /usr/src/app
+RUN apt-get update
```

Comment:
Combine apt-get update and apt-get install into one layer. [See this](https://docs.docker.com/develop/dev-best-practices/) and [this](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) for explanation.